src/server/server.cc

```diff
@@ -44,6 +44,7 @@
 #include "worker.h"
 
 std::atomic<int> Server::unix_time_ = {0};
+constexpr const char *REDIS_VERSION = "4.0.0";
```

Comment:
Out of curiosity, does it mean that we're functionality compatible with Redis 4.x?

Reply:
But we support STREAM delivered in Redis 5.

Reply:
I also think in the future there will be some commands that only kvrocks has but redis doesn't